src/Services/BusTrackerAPI.test.ts

```diff
@@ -1,24 +1,49 @@
 import axios from "axios";
 import BusTrackerAPI from './busTrackerAPI';
-const sampleResponse: JSON = require('./__fixtures__/searchResponse.json');
+const predictionResponse: JSON = require('./__fixtures__/getPredictions.json');
+const patternResponse: JSON = require('./__fixtures__/getPatterns.json');
+const routesResponse: JSON = require('./__fixtures__/getRoutes.json');
+const stopsResponse: JSON = require('./__fixtures__/getStops.json');
+const directionsResponse: JSON = require('./__fixtures__/getDirections.json');
```

Comment:
I'm pretty sure you can use import statements here, right?  `import predictionResponse from './__fixtures__/getPredictions.json'`?